### ISSUE TITLE
No module named 'wtforms.compat'

### ISSUE THREAD

**Issue by jnguiot** [kvesteri/wtforms-alchemy#158]
Versions:
- WTForms-Alchemy 0.17.0
- flask-wtf 1.0.0
-  WTForms 3.0.0
- WTForms-Components 0.10.5
- alembic 1.7.4

Observed behavior: unable to create a form with wtforms_alchemy
Expected behavior: able to create a form with wtforms_alchemy

Comment:
It seems like the recent WTForms 3.0.0 dropped "compat" module in it. But wtforms_alchemy is import it.

Thus, these errors happen:
```
tests/conftest.py:8: in <module>
    from collectives import create_app, auth
collectives/__init__.py:23: in <module>
    from . import models, api, forms
collectives/forms/__init__.py:14: in <module>
    from .auth import LoginForm, AccountCreationForm
collectives/forms/auth.py:10: in <module>
    from wtforms_alchemy import ModelForm
/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/wtforms_alchemy/__init__.py:21: in <module>
    from .fields import (  # noqa
/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/wtforms_alchemy/fields.py:11: in <module>
    from wtforms.compat import string_types, text_type
E   ModuleNotFoundError: No module named 'wtforms.compat'
```

The commit removing compat is https://github.com/wtforms/wtforms/commit/a34eb532d3b96ed216f204ed3d22fc9962241446

**Comment by rsyring** [comment]
FWIW, this issue is blocking a WTForms update which prevents addressing a security issue due to an XSS vulnerability in WTForms <3.0.0a1.  

https://github.com/wtforms/wtforms/commit/8529b953a0919300794f95e01a77131629

It's been added to the [Safety database](https://github.com/pyupio/safety-db) and is causing our CI to fail.

@kvesteri can you comment on whether or not this is something that the maintainers can address anytime soon, if you are open to contributions to get this fixed, and/or if the work started in #154 by @jpic is headed in the right direction?

Thanks.

**Comment by kvesteri** [comment]
Yes definitely open to contributions for getting this fixed. #154 is a step in the right direction

**Comment by rsyring** [comment]
Thanks for the quick reply.  @guruofgentoo, whose on my leadership team, is working to see if we can fit this into some of our contract work since almost all our clients use this library and would benefit from the enhancement.

**Comment by rsyring** [comment]
Fixed by @kvesteri in https://github.com/kvesteri/wtforms-alchemy/commit/ef0c1d5521e1d576734adbfe5fead8bb57fe0fba and released in 0.18.0.

Thank you.
